love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.jsx

```diff
@@ -246,14 +246,23 @@ export default class ConfigPanel extends Component {
    * @param {string} newValue, the new value to validate
    * @param {boolean} noRevalidation, if true, do not revalidate if already validating
    */
-  validateConfig = (newValue, noRevalidation) => {
+  validateConfig = (newValue, noRevalidation, postValidate=()=>{}, failValidate=()=>{}) => {
     this.setState({ value: newValue });
     /** Do nothing if schema is not available
      * stay in EMPTY state
      */
     const schema = this.props.configPanel.configSchema;
     if (!schema) {
-      this.setState({ validationStatus: EMPTY });
+      this.setState({
+        validationStatus: EMPTY,
+        configErrorTitle: 'Waiting for schema',
+        configErrors: [
+          {
+            name: ``,
+            message: `Schema not yet loaded`,
+          },
+        ],
+      });
```

Comment:
Are we 100% sure this will work with scripts that have no schema?

Reply:
I have seen that when a script does not have a schema, a `# empty Schema` is received. The implementation has initial state validationStatus: EMPTY, and is used to display the `...Waiting for Schema` message from the Schema read-only field. Verified in the script AuxTel/Calibration/power_off_atcalsys.py with get_schema() None, the API call responds with `# empty Schema` leaving a state other than validationStatus: EMPTY and the message `Waiting for Schema`